### PR TITLE
RDM-13045 Trigger integration and functional tests from CI pipeline

### PR DESCRIPTION
**Pull request by sem-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDM-13045

### Change description ###
Configure Jenkins pipeline to run the case-disposer integration and functional tests

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/ccd-case-disposer/values.preview.template.yaml [hunk 1]
```diff
@@ -46,13 +41,107 @@ logstash:
...
+        IDAM_DATA_STORE_SYSTEM_USER_PASSWORD: ${IDAM_DATA_STORE_SYSTEM_USER_PASSWORD}
+        IDAM_OAUTH2_DATA_STORE_CLIENT_SECRET: ${IDAM_OAUTH2_DATA_STORE_CLIENT_SECRET}
+      keyVaults: [ ]
+      autoscaling:
+        enabled: false
+
+  postgresql:
+    service:
+      type: LoadBalancer
+      annotations:
```
Comment (timja) [inline charts/ccd-case-disposer/values.preview.template.yaml:146]:
```suggestion
      annotations:
        external-dns.alpha.kubernetes.io/hostname=postgresql-${SERVICE_NAME}.
```

File: charts/ccd-case-disposer/values.preview.template.yaml [hunk 2]
```diff
@@ -46,13 +41,108 @@ logstash:
...
+        external-dns.alpha.kubernetes.io/hostname=postgresql-${SERVICE_NAME}.
```
Comment (timja) [inline charts/ccd-case-disposer/values.preview.template.yaml:147]:
```suggestion
        external-dns.alpha.kubernetes.io/hostname: postgresql-${SERVICE_NAME}.
```

File: charts/ccd-case-disposer/values.preview.template.yaml [hunk 3]
```diff
@@ -46,13 +41,108 @@ logstash:
...
+        IDAM_OAUTH2_DATA_STORE_CLIENT_SECRET: ${IDAM_OAUTH2_DATA_STORE_CLIENT_SECRET}
+      keyVaults: [ ]
+      autoscaling:
+        enabled: false
+
+  postgresql:
+    service:
+      type: LoadBalancer
+      annotations:
+        external-dns.alpha.kubernetes.io/hostname: postgresql-${SERVICE_NAME}.
```
Comment (timja) [inline charts/ccd-case-disposer/values.preview.template.yaml:147]:
```suggestion
        external-dns.alpha.kubernetes.io/hostname: postgresql-${SERVICE_FQDN}.
```

**Comment by sem-hmcts** [comment]
Closing this PR per PlatOps request

**Pull request closed without merging**